x-pack/elastic-agent/docs/inputs/input-logfile.asciidoc

```diff
@@ -0,0 +1,163 @@
+:type: logfile
+
+[id="{beatname_lc}-input-{type}"]
+=== Logfile input
+
+++++
+<titleabbrev>Logfile</titleabbrev>
+++++
+
+Use the `logfile` input to read lines from log files.
+
+To configure this input, specify a list of glob-based <<input-paths,`paths`>>
+that must be crawled to locate and fetch the log lines.
+
+Example configuration:
+
+["source","yaml",subs="attributes"]
+----
+inputs:
+- type: logfile
+  streams:
+    - paths:
+      - /var/log/message
+      - /var/log/*.log
+----
+
+
+You can apply additional
+<<{beatname_lc}-input-{type}-options,configuration settings>> (such as `fields`,
+`include_lines`, `exclude_lines`, `multiline`, and so on) to the lines harvested
+from these files. The options that you specify are applied to all the files
+harvested by this input.
+
+To apply different configuration settings to different files, you need to define
+multiple stream or input sections:
+
+["source","yaml",subs="attributes"]
+----
+inputs:
+  - type: logfile
+    streams:
```

Comment:
More general question: In standalone mode, should we encourage users to use multiple streams or multiple inputs? I'm thinking we should tell them to user multiple inputs as it is less hierarchy to configure.

It means, the `streams` concept does not even show up.

Reply:
This is indeed a good point, with that stream would be a fleet only concept, do we even need to document it?

Reply:
I guess it will be used for the system metrics input and will be indirectly documented there. But we don't need it for logs.